docs/platform/contracts/create.md

```diff
@@ -0,0 +1,20 @@
+**Usage**: `sdk.platform.contracts.create(contractDefinitions, identity)`    
+**Description**: This method will return a Contract object initialized with the parameters defined and apply to the used identity. 
+
+Parameters: 
+
+| parameters               | type              | required           | Description                                                       |  
+|--------------------------|-------------------|------------------	| -----------------------------------------------------------------	|
+| **contractDefinitions**  | JSONDataContract  | yes                | The defined [JSON Application Schema](https://dashplatform.readme.io/docs/explanation-platform-protocol-data-contract) |
+| **identity**             | Identity          | yes                | A valid [registered identity](/platform/identities/register.md) |
```

Comment:
```suggestion
| **identity**             | Identity          | yes                | A valid [registered `application` identity](/platform/identities/register.md) |
```

Reply:
Make sense, I added a task to the board to ensure we do that verification in the code too (by type)